ios/Video/NowPlayingInfoCenterManager.swift

```diff
@@ -176,6 +177,21 @@ class NowPlayingInfoCenterManager {
             }
             return .commandFailed
         }
+
+        // Handler for togglePlayPauseCommand, sent by Apple's Earpods wired headphones
+        togglePlayPauseTarget = remoteCommandCenter.togglePlayPauseCommand.addTarget { [weak self] _ in
+            guard let self, let player = self.currentPlayer else {
+                return .commandFailed
+            }
+
```

Comment:
```
    //Additional check: Is the player ready to play?
    guard player.currentItem?.status == .readyToPlay else {
        return .commandFailed
    }
```
@KrzysztofMoch 
WDYT?

Reply:
This is same code than play and pause handling. Looks good to me